tests/ingest_tester.py

```diff
@@ -348,7 +377,10 @@ def test_filler_chemical_names(runner, expected_names=None):
     print("Expected Filler Chemical Names found")
 
 
+# TODO Reimplement
+disabled.append("test_filler_trade_names")
 def test_filler_trade_names(runner, expected_names=None):
```

Comment:
https://stackoverflow.com/questions/2066508/disable-individual-python-unit-tests-temporarily
@unittest.skip

Reply:
I'll try that again, but I knew about that and gave it a shot, it didn't seem to work though for some reason

Reply:
You can also use the second solution in that StackOverflow post, which nops the code.

Reply:
My issue there is that it will still run the setup for the test, which is the slow part and I was hoping to skip it if possible to speed up testing.  So I suppose the options are to give up that benefit and run the test setup while disabling the test or to just go through the tests and comment out/remove references to the disabled tests, I can't think of a clean way to skip test setup with a decorator.  Maybe use the @disabled decorator and keep the disabled list, so at least this keeps the test function itself intact?

Reply:
I think I figured out how to do it with a decorator, assuming the CI passes